Modules/IO/JPEG2000/src/itkJPEG2000ImageIO.cxx

```diff
@@ -360,10 +360,7 @@ JPEG2000ImageIO::ReadImageInformation()
     this->m_Internal->m_Dinfo = nullptr;
   }
 
-  if (l_image)
-  {
-    opj_image_destroy(l_image);
-  }
+  opj_image_destroy(l_image);
```

Comment:
Cool, looks like the `if (!l_image)` at line 261 is indeed sufficient 👍

Reply:
Thanks for going over everything carefully!  I didn't trust cppcheck blindly, but more eyes always helps!